crates/bevy_render/src/camera.rs

```diff
@@ -232,11 +232,15 @@ impl NormalizedRenderTargetExt for NormalizedRenderTarget {
                     scale_factor: window.resolution.scale_factor(),
                 }),
             NormalizedRenderTarget::Image(image_target) => {
-                let image = images.get(&image_target.handle)?;
-                Some(RenderTargetInfo {
-                    physical_size: image.size(),
-                    scale_factor: image_target.scale_factor.0,
-                })
+                if let Some(image) = images.get(&image_target.handle) {
+                    Some(RenderTargetInfo {
+                        physical_size: image.size(),
+                        scale_factor: image_target.scale_factor.0,
+                    })
+                } else {
+                    error!("ImageRenderTarget handle unloaded. Make sure the Image's usages include RenderAssetUsages::MAIN_WORLD");
```

Comment:
Can we provide better information in this error about *what* handle you screwed up?

Reply:
Yeah, asset id is better than nothing.